### PR TITLE
Bugfix: interface transparency causing app logic inconsistencies

### PR DESCRIPTION
**Pull request by Feerip**
Turns out the issue was that the RGB value for the purple that Gorvek's name shows up in the chatbox was *slightly* miscalibrated. Changing it to the [correct value](https://github.com/Feerip/SusAlert/blob/cf489eae74652cbf75cb1ffce6a6c80ade87a748/scripts/script.js#L58) fixed the issue completely for every interface transparency below ~95%, and the rest of the colors I added (all of the colors I could think of that consistently showed up in a chatbox in a Croesus fight) were to make the consistency bulletproof, to the point that I couldn't get it to fail even at 100% interface transparency. 

Correct me if I'm wrong, but now that Gorvek's name is consistently recognized, it can be used as the anchor point/guaranteed reference point for state change messages in the fight, so that multi-line splicing business you had going on [here](https://github.com/Raphire/SusAlert/blob/ea5f04b802d291d5af1e63168677f59a9710dca4/scripts/script.js#L152) is no longer necessary, so I removed it, and reverted back to single-line-based detection. Please feel free to correct me if I'm wrong :)

### CODE REVIEW

**Comment by Raphire** [comment]
Heya, thanks for taking the time to look at that function. I haven't had much time to test/play with your changes. It however seems that there is an issue, in my testing all messages seem to come up as undefined. Can you confirm it's working on your end?

**Comment by Feerip** [comment]
I could have sworn I tested it to work, but apparently not, I see the errors too. My apologies, this is the first time I've used Javascript, my monkey brain is still trying to figure it all out. I'm troubleshooting it now.

**Comment by Raphire** [comment]
No worries! I can't say this hasn't happened to me ever before 😂

**Review by Feerip — COMMENTED** [review]
See below:

File: scripts/script.js [hunk 1]
```diff
@@ -164,24 +164,26 @@ function readChatbox()
 {
   var opts = chatReader.read() || [];
 
-  if (opts[0] != null)
+  for (var line in opts)
```
Comment (Feerip) [inline scripts/script.js]: Found it, this is the commit that broke everything. So it looks like I'm definitely misunderstanding the return value of chatReader.read(). Is there documentation anywhere on how to use alt1 calls like this? Maybe I'm just being a monkey but no matter how much googling I do I can't find anything. I've only gotten this far by brute forcing my way through it lol.

**Comment by Feerip** [comment]
So it turns out I just don't know how to use `for...in` loops with arrays in JS. It's fixed now, I tested it (after I was done this time 😂) and it works on my side. Please confirm from your side as well :)

**Comment by Raphire** [comment]
Hey, glad you found the issue . I unfortunately have been quite busy the past few days and will continue to be throughout the week. I'll be testing it out as soon as I get the chance. Thanks!

**Comment by Feerip** [comment]
No problem at all. Let me know if there's anything else I can do to help!

**Comment by Raphire** [comment]
Sorry for the long wait. All seems to be working great now! I approved the merge with some small changes to the readme.

**Pull request merged**